openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -3954,6 +3955,7 @@ public static Settings getSettings(SettingsType configType, String json) {
           switch (configType) {
             case EMAIL_CONFIGURATION -> JsonUtils.readValue(json, SmtpSettings.class);
             case CUSTOM_LOGO_CONFIGURATION -> JsonUtils.readValue(json, LogoConfiguration.class);
+            case CUSTOM_UI_THEME_PREFERENCE -> JsonUtils.readValue(json, UiThemePreference.class);
```

Comment:
we should merge logo and custom_ui into one object and write migrations to move that data. Don't want two separate configs for this.

Reply:
yes @harshach ,  custom_ui has both the logo and theme configurations, once the logo config usaage removed from backend, we can go with custom_ui config

Reply:
lets do it in this PR right. Instead of having it in separate.